strings/br.py

```diff
@@ -0,0 +1,433 @@
+
+# Part of the translation by https://github.com/DarrenWestwood
+
+# Currency symbol
+currency_symbol = "R$"
+
+# Positioning of the currency symbol
+currency_format_string = "{symbol} {value}"
+
+# Quantity of a product in stock
+in_stock_format_string = "{quantity} Disponível"
+
+# Copies of a product in cart
+in_cart_format_string = "{quantity} no Carrinho"
+
+# Product information
+product_format_string = "<b>{name}</b>\n" \
+                        "{description}\n" \
+                        "{price}\n" \
+                        "<b>{cart}</b>"
+
+# Order number, displayed in the order info
+order_number = "Order #{id}"
+
+# Order info string, shown to the admins
+order_format_string = "by {user}\n" \
+                      "Criado em {date}\n" \
+                      "\n" \
+                      "{items}\n" \
+                      "TOTAL: <b>{value}</b>\n" \
+                      "\n" \
+                      "Cliente notas: {notes}\n"
+
+# Order info string, shown to the user
+user_order_format_string = "{status_emoji} <b>Order {status_text}</b>\n" \
+                           "{items}\n" \
+                           "TOTAL: <b>{value}</b>\n" \
+                           "\n" \
+                           "Notas: {notes}\n"
+
+# Transaction page is loading
+loading_transactions = "<i>Carregando transações...\n" \
+                       "Por favor aguarde alguns segundos.</i>"
+
+# Transactions page
+transactions_page = "Página <b>{page}</b>:\n" \
+                    "\n" \
+                    "{transactions}"
+
+# transactions.csv caption
+csv_caption = "Um arquivo  📄 .csv contem todas transações guardadas no banco de dados do bot foi gerado.\n" \
+              "Você pode abrir um arquivo com outros programas, como LibreOffice Calc, Excel para processar" \
+              " os dados."
+
+# Conversation: the start command was sent and the bot should welcome the user
+conversation_after_start = "Olá!\n" \
+                           "Bem Vindo ao greed!\n" \
+                           "Isto é uma versão  🅱️ <b>Beta</b> do software.\n" \
+                           "É completamente usavel, mas pode haver bugs ainda presente.\n" \
+                           "If you find any, please report them at https://github.com/Steffo99/greed/issues."
+
+# Conversation: to send an inline keyboard you need to send a message with it
+conversation_open_user_menu = "O que você gostaria de fazer?\n" \
+                              "💰 Você tem <b>{credit}</b> na sua carteira.\n" \
+                              "\n" \
+                              "<i>Pressione uma tecla no teclado abaixo para selecionar uma operação.\n" \
+                              "Se o teclado não abriu, você pode abri-lo pressionando o botão com quatro  quadrados" \
+                              " pequenos na barra de menssagem.</i>"
+
+# Conversation: like above, but for administrators
+conversation_open_admin_menu = "Você é um  💼 <b>Gerente</b> desta loja!\n" \
```

Comment:
```suggestion
conversation_open_admin_menu = "Você é um 💼 <b>Gerente</b> desta loja!\n" \
```

There's an extra space here.